skunk.gemspec

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubycritic", "~> 4.0"
```

Comment:
@matiasgarcia RubyCritic moves pretty fast so I would prefer it if this dependency is a little more open. If we keep it like this, we will need to update it when they release 4.5+ versions 

What if you improve that to be >= 4.4 and < 5.0?

Reply:
Good to go I believe!